dynawo/cmake/FindSundials.cmake

```diff
@@ -65,6 +65,7 @@ if (SUNDIALS_INCLUDE_DIR AND SUNDIALS_IDA_LIBRARY)
     "int main() {\n"
     "  void* IDAMem = IDACreate();\n"
     "  int flag = IDASetMinStep(IDAMem, 0.1);\n"
+    "  static_cast<void>(flag);"
```

Comment:
Why is this needed?

Reply:
to use the flag variable and avoid a warning as error in some cases